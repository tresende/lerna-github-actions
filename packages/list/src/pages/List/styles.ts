import styled from 'styled-components'

export const Wrapper = styled.main`
  padding: 2rem;
  margin: 0 auto;
`
