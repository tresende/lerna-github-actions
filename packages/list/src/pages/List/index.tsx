import React from 'react'

import Table from '../../components/Table'

import * as S from './styles'

const List = () => (
  <S.Wrapper>
    <Table />
  </S.Wrapper>
)

export default List
