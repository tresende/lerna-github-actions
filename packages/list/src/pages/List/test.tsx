import React from 'react'
import { render } from '@testing-library/react'

import List from '.'

describe('<List />', () => {
  it('should ...', () => {
    const { container } = render(<List />)
    expect(1).toBe(1)
    expect(container.firstChild).toMatchSnapshot()
  })
})
