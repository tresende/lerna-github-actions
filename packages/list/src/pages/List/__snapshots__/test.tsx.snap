// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<List /> should ... 1`] = `
<div
  class="sc-bdvvtL"
>
  <h1>
    List
  </h1>
</div>
`;
